spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+<emu-clause id="pluralrules-objects">
+  <h1>PluralRules Objects</h1>
+
+  <emu-clause id="sec-intl-pluralrules-abstracts">
+    <h1>Abstract Operations for PluralRules Objects</h1>
+
+    <emu-clause id="sec-initializepluralrules" aoid="InitializePluralRules">
+      <h1>InitializePluralRules ( _pluralRules_, _locales_, _options_ )</h1>
+
+      <p>
+        The abstract operation InitializePluralRules accepts the arguments _pluralRules_ (which must be an object), _locales_, and _options_. It initializes _pluralRules_ as a PluralRules object. The following steps are taken
+      </p>
+
+      <emu-alg>
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. If _options_ is *undefined*, then
+          1. Let _options_ be ObjectCreate(*null*).
+        1. Else
+          1. Let _options_ be ? ToObject(_options_).
+        1. Let _t_ be ? GetOption(_options_, `"type"`, `"string"`, &laquo; `"cardinal"`, `"ordinal"` &raquo;, `"cardinal"`).
+        1. Set _pluralRules_.[[Type]] to _t_.
+        1. Let _opt_ be a new Record.
+        1. Let _matcher_ be ? GetOption(_options_, `"localeMatcher"`, `"string"`, &laquo; `"lookup"`, `"best fit"` &raquo;, `"best fit"`).
+        1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. Perform ? SetNumberFormatOptions(_pluralRules_, _options_, *0*, *3*).
+        1. Let _r_ be ResolveLocale(%PluralRules%.[[AvailableLocales]], _requestedLocales_, _opt_).
+        1. Set _pluralRules_.[[Locale]] to the value of _r_.[[Locale]].
+        1. Let _pluralCategories_ be a List of Strings representing the possible results of plural selection.
+        1. Set _pluralRules_.[[PluralCategories]] to CreateArrayFromList(_pluralCategories_).
+        1. Return _pluralRules_.
+      </emu-alg>
+
+    </emu-clause>
+
+    <emu-clause id="sec-getoperands" aoid="GetOperands">
+       <h1>GetOperands ( _s_ )</h1>
+       <p>
+         When the GetOperands abstract operation is called with argument _s_. It performs the following steps:
+       </p>
+
+       <emu-alg>
+         1. Assert: Type(_s_) is String.
+         1. Let _n_ be ? ToNumber(_s_).
+         1. Assert: _n_ is finite.
+         1. Let _dp_ be ? Call(%StringProto_indexOf%, _s_, &laquo; `"."` &raquo;).
+         1. If _dp_ = -1, then
+           1. Set _iv_ to _n_.
+           1. Let _f_ be 0.
+           1. Let _v_ be 0.
+         1. Else,
+           1. Let _iv_ be the substring of _s_ from position 0, inclusive, to position _dp_, exclusive.
+           1. Let _fv_ be the substring of _s_ from position _dp_, exclusive.
+           1. Let _f_ be ? ToNumber(_fv_).
+           1. Let _v_ be ? ToLength(? Get(_fv_, `"length"`)).
+         1. Let _i_ be ? abs(? ToNumber(_iv_)).
```

Comment:
`abs` is a [mathematical operation](https://tc39.github.io/ecma262/#sec-mathematical-operations), so it should neither be preceded by `?` nor by `!`

/? ToNumber/! ToNumber/